Modules/src/LED.cpp

```diff
@@ -12,13 +13,36 @@ namespace STM32LED
         this->LED_state = state::off;   //default state is off
     }
 
+    //checks current state - if 
     void LED::set(state s)
     {
+        if (s == state::on)
+        {
+            LED_state = state::on;

```

Comment:
In order to access an object within the class, use `this->` 
```suggestion
            this -> LED_state = state::on;

```